sqldb/sqlc/migrations/000001_invoices.up.sql

```diff
@@ -1,7 +1,17 @@
+-- sequences contains all sequences used for invoices.
+CREATE TABLE invoice_sequences (
```

Comment:
Rationale for this route instead of using the existing auto increment feature of primary keys? Is it that if used incorrectly they can actually be non-contiguous, or other?